packages/myst-cli-utils/tsconfig.json

```diff
@@ -4,7 +4,7 @@
     // module is overridden from the build:esm/build:cjs scripts
     "module": "es2015",
     "jsx": "react-jsx",
-    "lib": ["es2020"],
+    "lib": ["es2021"],
```

Comment:
This is necessary to use the `replaceAll` string method (introduced in node 15).